lib/fetchers/TokenFetcher.ts

```diff
@@ -38,7 +38,7 @@ export class TokenFetcher {
    * Gets the token address for the provided token symbol or address from the DEFAULT_TOKEN_LIST.
    * Throws an error if the token is not found.
    */
-  public resolveTokenAddress = async (chainId: ChainId, symbolOrAddress: string): Promise<string> => {
+  public resolveTokenSymbolOrAddress = async (chainId: ChainId, symbolOrAddress: string): Promise<string> => {
```

Comment:
v nit but the function name reads as if it'll return either a token symbol or address

Reply:
ah i see okay maybe `resolveTokenBySymbolOrAddress `

Reply:
changed to this name